firefox-ios/Client/AccessoryViewProvider.swift

```diff
@@ -199,7 +199,7 @@ class AccessoryViewProvider: UIView, Themeable {
         toolbar.accessibilityElements = [
             previousButton,
             nextButton,
-            currentAccessoryView,
+            currentAccessoryView?.customView,
```

Comment:
Since this is a change in the app itself, I would ping a dev to approve this change.